From 351255dd5ed00a281da97a6ac84c386d42ce19df Mon Sep 17 00:00:00 2001
From: "Brian W. Hart" <hartb@us.ibm.com>
Date: Mon, 4 May 2020 19:51:31 +0000
Subject: [PATCH] Fix spelling of skipTest()

The unittest module's exception for skipping tests is capitalized
("SkipTest") but the TestCase class's method for skipping is not
("skipTest"). Fix the spelling in TestCaffe2Export._get_test_image()
to avoid:

AttributeError: 'TestCaffe2Export' object has no attribute 'SkipTest'
---
 tests/test_export_caffe2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_export_caffe2.py b/tests/test_export_caffe2.py
index 31454ae..ad989c4 100644
--- a/tests/test_export_caffe2.py
+++ b/tests/test_export_caffe2.py
@@ -49,7 +49,7 @@ class TestCaffe2Export(unittest.TestCase):
             file_name = DatasetCatalog.get("coco_2017_train")[0]["file_name"]
             assert PathManager.exists(file_name)
         except Exception:
-            self.SkipTest("COCO dataset not available.")
+            self.skipTest("COCO dataset not available.")
 
         with PathManager.open(file_name, "rb") as f:
             buf = f.read()
-- 
1.8.3.1

